distribution/docker/ddf/src/main/docker/files.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0">
```

Comment:
✏️  This includes resources into something? I'm guessing it copies the files needed for a Solr core into an artifact, but it is hard for me to tell.

Reply:
This includes resources into the docker build context so that instructions in the dockerfile can actually use them.